webapp/src/components/Activity.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+interface ActivityProps {
```

Comment:
Should we add this to the api.d.ts file?

Reply:
Good question! I don't think so. I'm not sure we're going to end up keeping this file anyway, but it seems the standard we have in other files is to have the Props structure in-file with the component.